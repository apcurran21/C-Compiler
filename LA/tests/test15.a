void main ( ){

  int64[][][] a
  a <- new Array(2,3,4)
   print(a)
   initArray(a)
  return 

}

void initArray (int64[][][] A){

  A[1][2][4] <- 0
  return

}
