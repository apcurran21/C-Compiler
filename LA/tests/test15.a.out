{s:27, 2, 3, 4, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0}
ERROR: instruction at line 13 attempted to use position 4 in the dimension 2 of a tensor that only has 4 elements in that dimension
